workspace-manager-clienttests/README.md

```diff
@@ -16,16 +16,25 @@ Note: `render-config.sh` assumes a valid `.vault-token` in your `$HOME` director
 ./render-config.sh
 ```
 
-To run the test, use the following command from the workspace-manager-clienttests directory.
+To run the tests, use one of the following commands from the workspace-manager-clienttests directory.
+NB: The `BasicIntegration.json` and `BasicNightlyPerf.json` are sample test suite configurations for Integration and Perf tests.
+See `test-runner-integration.yml` and `test-runner-nightly-perf.yml` in `.github/workflows` for use cases.
 
+Integration Test
 ```
-./gradlew  runTest --args="configs/BasicAuthenticated.json /tmp/TR"
+./gradlew  runTest --args="suites/BasicIntegration.json /tmp/TR"
```

Comment:
I think the README should include both instructions: how to run an individual test and how to run a suite. For developers writing tests, it will often be helpful to run a single test.

Reply:
Both individual test and test suite modes are included in README now.